integration-tests/utils/integration-utils.js

```diff
@@ -85,7 +85,7 @@ const select = async (page, labelContains, text) => {
   return click(page, `//div[starts-with(@id, "react-select-") and contains(normalize-space(.),"${text}")]`)
 }
 
-const waitForNoSpinners = page => {
+const waitForNoSpinners = async (page) => {
```

Comment:
Code bug: missing `async` modifier.

Reply:
So, because the body of the function doesn't contain the keyword `await`, this shouldn't be marked `async`. The function just returns the promise from the `waitForXPath` directly, which happens to be async, but this function doesn't need to care.

Reply:
Yes, I knew. I think the modifier makes function clear and syntactically correct. I've using typescript, without the modifier, it looked "odd". I'll remove the modifier.

Reply:
Calling `waitForNoSpinners(page)` function without `await` could have been prevented at compile time if `async` is present in function declaration.